### DISCUSSION TITLE
Any requirements for Gradle and/or flutter and/or JDK versions to make patrol 2.0.0 work?

### DISCUSSION THREAD

**Discussion by EyedBread** [leancodepl/patrol#1421] [Q&A] [answered]
Currently trying to integrate Patrol 2.0.0 into our flutter app and are facing some issues. Don't want to bore you with all the details but we have a lot of dependencies and packages and are currently using flutter 3.7.12 and gradle 7.4.2 with jdk 8 and are having trouble following the native automation setup https://patrol.leancode.co/getting-started/native due to not being able to execute the ```patrol test --target integration_test/example_test.dart```. I've tried looking for any minimal requirements for patrol 2.0.0 but can't seem to find it.

**Comment by bartekpacia** [comment, marked as answer]
Hi, sorry for the problems. Patrol doesn't have any specific requirements besides Flutter.

What is probably wrong with your setup is the JDK version. A minimal version of 11 is required, and you have 8. JDK11 is also required by Flutter 3.10.x AFAIK.

We currently support Flutter 3.3 and newer, but I highly recommend to always use the latest version.

Gradle 7 is fine, though again, Gradle 8 is better B) 

It'd help a lot if you shared the (verbose!) logs, it's hard to tell what's the specific problem you're having.
